### ISSUE TITLE
IPR037062 IPR012301 IPR001891 IPR015884

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#4728]
* **InterPro ID / label**
IPR037062 Malic enzyme, N-terminal domain superfamily
IPR012301 Malic enzyme, N-terminal domain
IPR001891 Malic oxidoreductase
IPR015884 Malic enzyme, conserved site

* **Example sequences with problematic annotation (ID + gene/protein name):**
FBgn0034127	CG7848
FBgn0002719	Men
FBgn0029155	Men-b
FBgn0029154	Menl-1
FBgn0029153	Menl-2

* **Description of issue**
All four of these InterPro entries are annotated with the NAD-specific GO MF term "malate dehydrogenase (decarboxylating) (NAD+) activity (GO:0004471)" (= EC:1.1.1.38 /EC:1.1.1.39)

But all four entries have descriptions saying  "There are 3 forms of the enzyme: an NAD-dependent form that decarboxylates oxaloacetate; an NAD-dependent form that does not decarboxylate oxalo-acetate; and an NADPH-dependent form"
Consistent with that, all four entries are associated with these 3 ECs:
1.1.1.38 malate dehydrogenase (oxaloacetate-decarboxylating) = GO:0004471
1.1.1.39 malate dehydrogenase (decarboxylating) = GO:0004471
1.1.1.40 malate dehydrogenase (oxaloacetate-decarboxylating) (NADP(+)) = GO:0004473

That is, all four signatures may be associated with GO:0004471 (malate dehydrogenase (decarboxylating) (NAD+) activity) or GO:0004473 (malate dehydrogenase (decarboxylating) (NADP+) activity)

So I think the GO:0004471 annotation on all four entries should be replaced with the shared parent terms of GO:0004471 & GO:0004473:
GO:0004470 malic enzyme activity
GO:0016616 oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor

**Comment by sarach06** [comment]
Hi Steven, 
Thank you very much for this. I have updated the annotations as you suggested for IPR037062 Malic enzyme, N-terminal domain superfamily, IPR012301 Malic enzyme, N-terminal domain and IPR001891 Malic oxidoreductase. In the conserved site entry, I removed the term as from the description, is not clear its function yet. Let me know if you agree or have any questions. This will be visible in InterPro 97.0.
Best regards,
Sara
